README.md

```diff
@@ -6,19 +6,21 @@ description: Introducing Subsquid, a GraphQL query node for substrate chains.
 
 ## Welcome
 
-Welcome to Subsquid documentation. The material here is meant to introduce the general public to  Subsquid query node framework and to be an asset at the developers' disposal while building with Subsquid.
+Welcome to Subsquid documentation. The material here is meant to introduce the general public to the Subsquid query node framework and to be an asset at the developers' disposal while building with Subsquid.
 
 The various sections and pages cover a variety of subjects, including concepts at the base of the project, operational guides, as well as reference material.
 
 ## What is Subsquid
 
 [Subsquid ](https://subsquid.io)is a query node framework for Substrate-based blockchains. In very simple terms, Subsquid can be thought of as an ETL tool, with a GraphQL server included.
 
-A query node ingests data from a substrate chain and provides rich, domain-specific, and highly customizable access to the blockchain data, far beyond the scope of direct RPC calls.
+Subsquid's [multi-layer approach](key-concepts/architecture.md) aims to pre-process and decode raw chain data and store it for easier access by the query nodes, providing increased performance over direct RPC calls.
+
+A Squid Archive can constantly synchronize raw data with the chain and by simply defining the custom data model and Event handlers in the Squid project, it is possible to perform complex queries on top of it.
 
 For example, expired [Kusama Treasury](https://wiki.polkadot.network/docs/en/learn-treasury) spending [proposals](https://kusama.subscan.io/event?module=Treasury\&event=Proposed) are pruned from the state of the [Kusama blockchain](https://polkascan.io/kusama), so querying one-year-old proposals would be problematic. With the traditional approach, it would be necessary to track the evolution of the state by sequentially applying the Treasury [events and extrinsics](key-concepts/substrate.md) in each historical block.
```

Comment:
See above

Reply:
I kind of agree. I restructured this part and after reading it, I placed it here "as an example", but might be a bit irrelevant.